rotkehlchen/db/upgrades/v46_v47.py

```diff
@@ -246,4 +247,24 @@ def _remove_old_accounting_rules(write_cursor: 'DBCursor') -> None:
             ],
         )
 
+    @progress_step(description='Remove manual historical oracle')
+    def _remove_manual_historical_price_oracle(write_cursor: 'DBCursor') -> None:
+        write_cursor.execute(
+            'SELECT value FROM settings WHERE name=?',
+            ('historical_price_oracles',),
+        )
+        result = write_cursor.fetchone()
+        if result is None:
+            return
+
+        try:
+            oracles = [x for x in json.loads(result[0]) if x != 'manual']
+        except json.JSONDecodeError:
+            oracles = [x.serialize() for x in DEFAULT_HISTORICAL_PRICE_ORACLES_ORDER]
```

Comment:
but you should also log an error here. So that by looking at the logs one can know the this happened.